### PR TITLE
sokol: add version 20230925

### PR DESCRIPTION
**Pull request by choll**
Specify library name and version:  **sokol/cci.20230925**

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by RubenRBS — APPROVED** [review]

File: recipes/sokol/all/conandata.yml [hunk 1]
```diff
@@ -2,3 +2,6 @@ sources:
   "cci.20210304":
     url: "https://github.com/floooh/sokol/archive/c5d55faa1036af9919bd28400e97d37548bb3789.zip"
     sha256: e61c782e287543ffb8077ac24a7bfee05e867ee27f6c3e9487a2dbfa1ca2b466
+  "cci.20230925":
+    url: "https://github.com/floooh/sokol/archive/5633a0f91bf3354320837ce0c7b4748a953a1401.zip"
+    sha256: 9c854bc7e6f73024a99554e8e1c8850fddfc325bb2b1ece2c59502c0e3cef68d
```
Comment (jwillikers) [inline recipes/sokol/all/conandata.yml:7]: Newer versions should be put first, at the top of the list. If you could make that change here and in the `config.yml` that would be great.

Comment (jwillikers) [reply recipes/sokol/all/conandata.yml:4]: Thanks!

**Review by jwillikers — APPROVED** [review]

File: recipes/sokol/all/conandata.yml (hunk 1 above)
Comment (choll) [reply recipes/sokol/all/conandata.yml:4]: I've just pushed a fix in https://github.com/conan-io/conan-center-index/pull/20250/commits/59cb4934404ea0d9d5c3f54a5f398c54a2b336ec, thanks

**Comment by RubenRBS** [comment]
Retriggered v1 pipeline, it found a transient network error, thanks a lot for your patience

**Review by RubenRBS — APPROVED** [review]

**Review by jwillikers — APPROVED** [review]

**Comment by choll** [comment]
@jwillikers apologies, I didn't realise hitting the "merge from master" button would clear your approval, could you approve again please? Thanks

**Review by jwillikers — APPROVED** [review]

**Pull request merged**